### DISCUSSION TITLE
Important Announcement: Code move to matter.js

### DISCUSSION THREAD

**Discussion by Apollon77** [mfucci/node-matter#274] [General]
We decided to join forces with the official Project CHIP as main home for the Matter standard! Because of this node-matter will be suspended and moved into https://github.com/project-chip/matter.js ! Please follow the new repository for updates!

We will keep this repository up for a while, but it will not be updated anymore as soon as the move has been finished and a new package "matter-node.js" has been deployed to npm initially.

We will also update documentation and try to move the discussions or such over as soon as we know what's possible. If you want to support by moving informations that are important from your PoV please support by just duplicating them over there. 

Please do not do any new code contributions to this project, but direct it to the new home!

**Comment by vves** [comment]
What is the new GitHub repo address?

**Reply by mfucci** [reply]
https://github.com/project-chip/matter.js

**Reply by mfucci** [reply]
The MR to move the code is not merged yet: https://github.com/project-chip/matter.js/pull/57

**Reply by Apollon77** [reply]
@vves I will soon moce over all open PRs too ... I will request your Review then there

**Comment by Hedda** [comment]
Any news on this move and PR https://github.com/mfucci/node-matter/pull/273 ?

**Reply by Apollon77** [reply]
Move is basically done already , I merge that PR now
